plugin/log/README.md

```diff
@@ -72,10 +73,10 @@ The following place holders are supported:
 * `{>do}`: is the EDNS0 DO (DNSSEC OK) bit set in the query
 * `{>id}`: query ID
 * `{>opcode}`: query OPCODE
-* `{local}`: server's IP address, for IPv6 addresses these are enclosed in brackets: `[::1]`
-* `{A}`: number of the A type answers in the response
-* `{AAAA}`: number of the AAAA type answers in the response
-* `{forward/resolving_proxy}`: if the query was forwarded to a resolver, the IP address and port of that replied resolver, else empty. The metadata plugin needs to be enabled to use this placeholder
+* `{<plugin>/<name>}`: metadata labels are accepted as place holders and will be replaced by the corresponding metadata value.
```

Comment:
for easy scaning these should start with a `/`

Reply:
The name of the labels are defined by any plugin providing the metadata, not by the Log. 
I use here the generic label format we defined (see metadata.IsLabel())

That will not change the scanning that look for "{" and "}" anyway.

Reply:
NOTE: maybe what you mean is that we could transform the standard placeholder of Log into metadata information that startwith "/".

eg : 
- {name} => could be a metadata /name and then used as {/name} in Log ?
- {type} => could be a metadata /type and then used as {/type} in Log ?

Reply:
yes, each metadata thingy starts with `/` for easy detection and make it more visible